source/services/dialog/dialog.service.ts

```diff
@@ -58,6 +46,50 @@ export class DialogService<TDialogSettings> implements IDialogService<TDialogSet
 
 		return this.dialog.prompt(options, require('./promptDialog.html'));
 	}
+
+	openAutosaveForm(options: IAutosaveDialogSettings): void {
+		this.promise.resolvePromises(options.resolve).then((resolveData: any): void => {
+			let scope: IAutosaveDialogScope = <IAutosaveDialogScope>options.scope;
+
+			if (scope == null) {
+				scope = <IAutosaveDialogScope>this.$rootScope.$new();
```

Comment:
Do we want to base this on root scope, or should we consider the $parent scope as the parent? Probably can wait until we see how this is used and / or how it needs to be used.